core/imodel-bridge/src/BADGERSIssueReporter.ts

```diff
@@ -0,0 +1,288 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { DbResult, Guid, GuidString } from "@bentley/bentleyjs-core";
+import { ECDb } from "@bentley/imodeljs-backend";
+import { BridgeIssueReporter } from "./BridgeIssueReporter"
+import * as fs from "fs";
+import * as path from "path";
+import * as os from "os";
+import { fips } from "crypto";
+
+interface ContextInfo {
+    activityId: string;
+    contextId: string;
+    iModelId: string;
+    briefcaseId: number;
+    operationType?: string;
+    jobId?: string;
+}
+
+interface FileInfo {
+    fileId?: string;
+    filePath?: string;
+}
+
+/** BADGERS issue reporter. Can be provided to BridgeRunner in setIssueReporter
+ * @beta
+ */
+export class BADGERSIssueReporter implements BridgeIssueReporter {
+    private _reportDb;
+    private _reportFile;
+    private _reportSummaryFile; //ToDo create summary report
+    private _stagingDir;
+    private _contextInfo: ContextInfo;
+    private _fileInfo: FileInfo;
+    private _dbPath: string = "";
+    // private _authHandler; //Callback to get token. Refer to serverArgs
+    // private _signInMgr;
+    // private _isValidContext;
```

Comment:
Is there a good reason to keep these comments around?

Reply:
Variables have been removed since it is now the responsibility of the orchestrator to upload reports